src/screens/notifications/notifications.tsx

```diff
@@ -90,20 +90,20 @@ export class NotificationsComponent extends React.Component<
     public async componentDidMount() {
         await LoadingModal.open();
```

Comment:
here you should call `fetchNotifications` and that's it. Everything else should be done there, even the modal open/close.